config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'about_us/index'
```

Comment:
update the way you define these routes

e..g.
`root controller: :products, action: :index`
`resources :about_us, only: [:index]`

Reply:
Thank you.
Updated ;)